validation/shallow_water_model/shallow_water_jet_topography.jl

```diff
@@ -0,0 +1,164 @@
+using Oceananigans
+using Oceananigans.Models: ShallowWaterModel
+   
+Nx, Ny = 50, 128
+Lx, Ly = 2π,   10
+H₀     = 10
+α, β   = 0.1, -0.1
+Lb, Hb = α * Ly, β * H₀
+
+f, g, U  = 1, 1, 1
+Δη = -f * U / g 
+
+grid = RectilinearGrid(size = ( Nx, Ny),
+                          x = (  0, Lx),
+                          y = (-Ly, Ly),
```

Comment:
it's a bit counter-intuitive that we define `Ly = 10` and then `grid.Ly = 20`

Reply:
can we change to

```
Ly = 20
...
grid = RectilinearGrid(..., y = (-Ly/2, Ly/2), ...)
```

Reply:
Yes, a much better idea!

Reply:
Done!